src/associations/base.d.ts

```diff
@@ -39,6 +39,8 @@ export interface ForeignKeyOptions extends ColumnOptions {
   name?: string;
 }
 
+export type OnDeleteUpdateAction = 'RESTRICT' | 'CASCADE' | 'NO ACTION' | 'SET DEFAULT' | 'SET NULL';
```

Comment:
Do we want to include this in this PR?

Reply:
No let's remove it from this PR and discuss it in another one